libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/utils/RouterEx.kt

```diff
@@ -15,3 +17,5 @@ fun com.mapbox.navigation.base.route.RouterOrigin.mapToNativeRouteOrigin(): Rout
         com.mapbox.navigation.base.route.RouterOrigin.Onboard -> RouterOrigin.ONBOARD
         is com.mapbox.navigation.base.route.RouterOrigin.Custom -> RouterOrigin.CUSTOM
     }
+
+fun NavigationRoute._waypoints(): List<Waypoint> = waypoints
```

Comment:
```suggestion
fun NavigationRoute.waypointsList(): List<Waypoint> = waypoints
```
nit

Reply:
added `_` to mark it kinda `internal`. So may be
```suggestion
fun NavigationRoute.internalWaypoints(): List<Waypoint> = waypoints
```